book.toml

```diff
@@ -1,5 +1,5 @@
 [book]
-authors = ["The Rust Project Developers"]
+authors = ["steveklabnik"]
```

Comment:
Haha :P I guess you've earned this but I think it would be nice to keep this as is ;)